### DISCUSSION TITLE
Metadata no longer displayed for OLAP cubes

### DISCUSSION THREAD

**Discussion by bdarbonneau** [DaxStudio/DaxStudio#971] [Q&A]
In previous versions of DAX studio, I could connect to OLAP cubes (v 11.0.6020.0) and the metadata pane would display all the artifacts of the cube. I do not exactly the version when this stopped working, but with the most recent one (3.0.5), the metadata pane is empty. I can run previously saved queries and get results, but I am unable to explore the tables, columns and measures of the cube.
Is there a setting that I am missing ? Is the support of OLAP cubes deprecated ?

**Comment by dgosbell** [comment]
I have not specifically deprecated anything. When you say "OLAP cubes" are you talking about a multidimensional cube? Do you see any errors in the log pane?

**Comment by bdarbonneau** [comment]
Hi Darren,

Yes I referred to multidim cubes.

I get either of these two messages in the log pane. I have set the log level to verbose.

06/10/2022 09:46:24                     The name of the object 'Table' cannot be the reserved string 'Measures'.
06/10/2022 09:48:56                     Error refreshing table list: The server sent an unrecognizable response.

is there a way to visualize the data exchanged back & forth with the server ?

**Reply by dgosbell** [reply]
I've installed a local multi-dim instance and can reproduce this issue. I've checked in a fix which will be part of the next release.

**Comment by bdarbonneau** [comment]
Hi Daren,
This is a very good news. Thanks for all your efforts to maintain this essential tool.
Regards,
Bertrand
